sklearn/compose/_column_transformer.py

```diff
@@ -51,20 +51,20 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
     Parameters
     ----------
     transformers : list of tuples
-        List of (name, transformer, column(s)) tuples specifying the
+        List of (name, transformer, columns) tuples specifying the
```

Comment:
I did not like the notation. `column(s)` in a tuple looked strange to me, like a function call  or something. I find `columns` and information that it can be a list or just a plain string way more readable. 

This is of course very subjective. If you prefer the original just let me know, I will revert this change.

Reply:
That's how I understood your change and I find it more readable as well.